sources/tests/Fixtures/PikaChuJuicer.php

```diff
@@ -38,12 +39,7 @@ protected function doTrimAndLowerString($value): string
     protected function mustNotBeEmptyString($value)
     {
         if (strlen($value) === 0) {
-            throw new ValidationException(
-                sprintf(
-                    "Field '%s' is an empty string.",
-                    $name
-                )
-            );
+            throw new ValidationException("Field 'pika' is an empty string.");
```

Comment:
I totaly agree with you and this is why I opened this issue https://github.com/chanmix51/ParameterJuicer/issues/3 but here the `$name` is not present and it's impossible to retrieve the tested field